openedx/core/djangoapps/notifications/views.py

```diff
@@ -279,16 +277,26 @@ def get(self, request):
         )
         count_total = 0
         count_by_app_name_dict = {}
+        show_notifications_tray_enabled = False
 
         for item in count_by_app_name:
             app_name = item['app_name']
             count = item['count']
-
             count_total += count
             count_by_app_name_dict[app_name] = count
-        # Return the unseen notifications count for the user and the unseen notifications count for each app name.
+
+        learner_course_enrollments = CourseEnrollment.objects.filter(
+            user=request.user,
+            is_active=True
+        ).values_list('course_id', flat=True)
+
+        for enrollment in learner_course_enrollments:
+            if SHOW_NOTIFICATIONS_TRAY.is_enabled(enrollment):
```

Comment:
I think learner_enrollments_course_ids would be better


`
learner_enrollments_course_ids = CourseEnrollment.objects.filter(
            user=request.user,
            is_active=True
        ).values_list('course_id', flat=True)

        for course_id in learner_enrollments_course_ids:
            if SHOW_NOTIFICATIONS_TRAY.is_enabled(course_id):
`